addons/survey/static/src/js/survey_form.js

```diff
@@ -46,6 +53,34 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     // Handlers
     // -------------------------------------------------------------------------
 
+    _onKeyPress: function (event) {
+        var self = this;
+        var aIndex = 'a'.charCodeAt(0);
+        var zIndex = 'z'.charCodeAt(0);
+        var code = event.keyCode;
+        var insideTextArea = this.$("textarea").is(":focus");
+        var $choiceInputs = this.$("input[type='radio'],input[type='checkbox']");
+        // Handle Start / Next / Submit
+        if (code === 13 && !insideTextArea) {  // Enter : go Next
+            event.preventDefault();
+            this.$("button.btn-primary").click();
+        } else if (!this.options.readonly
+                && self.options.questionsLayout === 'page_per_question'
+                && !insideTextArea
+                && aIndex <= code < zIndex && $choiceInputs.length && $choiceInputs.length < 26) {
+            var keyIndex = code - aIndex;
```

Comment:
Can't you put the key in an data attribute to try to find the corresponding one instead?
This is rather dirty.